src/features/apiDelegation/components/OverviewPageContent/OverviewPageContent.tsx

```diff
@@ -88,7 +89,8 @@ export const OverviewPageContent = ({
 
   const goToStartDelegation = () => {
     dispatch(restoreAllSoftDeletedItems());
-    navigate('/' + RouterPath.OfferedApiDelegations + '/' + RouterPath.ChooseOrg);
+    // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
```

Comment:
Why this disabling?
What is it complaining about that it wasn't before?
And can it be solved some other way?

Reply:
Ahh I forgot to change the old RouterPath.ChooseOrg to ApiDelegationPath.ChooseOrg